xmpp-vala/src/module/sasl.vala

```diff
@@ -3,19 +3,23 @@ using Xmpp.Core;
 namespace Xmpp.PlainSasl {
     private const string NS_URI = "urn:ietf:params:xml:ns:xmpp-sasl";
 
+    public interface PasswordProvider : GLib.Object {
+        public abstract async string? get_password();
```

Comment:
Shouldn’t there also be set_password here?

Reply:
It's not needed... so, I don't really care. This is purely for `PlainSasl`, which never changes the password.

Reply:
Maybe add a comment with the reason.